Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "shell"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = { version = "4.0", features = ["derive"] }
```

Comment:
This dependency is a fairly heavy dependency and doesn't seem to be used yet - can we remove this?